docs/qrcode.md

```diff
@@ -0,0 +1,186 @@
+
+```js
+import Airwallex from 'airwallex-payment-elements';
+```
+
+or add the bundle as a script in your HTML head
+
+```html
+<script src="https://checkout.airwallex.com/assets/bundle.x.x.x.min.js"></script>
+```
+
+Be sure to replace the x.x.x with the `airwallex-payment-elements` package version you'd like to use.
+
+### 2. Initialize the Airwallex package with the appropriate environment
+
+```js
+Airwallex.loadAirwallex({
+  env: 'demo', // Setup which Airwallex env('staging' | 'demo' | 'prod') to integrate with
+  origin: window.location.origin, // Setup your event target to receive the browser events message
+});
+```
+
+`loadAirwallex` takes in options to set up the payment environment. See docs for further customizations [here](/docs#loadAirwallex).
+
+The Airwallex package only needs to be mounted once in an application (and everytime the application reloads).
+
+### 3. Add an empty container for the qrcode element to be injected into
+
+```html
+<div id="qrcode"></div>
+```
+
+We will mount the qrcode element into the empty div in Step 5.
+
+### 4. Create the qrcode element
+
+This creates the specified [Element](/docs#Element) object. We specify the type as **`qrcode`**.
+
+```js
+const element = Airwallex.createElement('qrcode', {
+  intent: {
+    // Required, the qrcode element uses intent_id and client_secret to prepare checkout
+    id: 'replace-with-your-intent-id',
+    client_secret: 'replace-with-your-client-secret',
+  },
+  qrcodeMethod: 'replace-with-your-payment-method' // available methods: 'alipayhk', 'kakaopay', 'wechatpay'
+});
+```
+
+You **must provide intent details** to create the qrcode element.
+
+There are also additional options as a second parameter to the `createElement` function that can overwrite styles and other functions. [See docs](/docs#createElement) for more details.
+
+### 5. Mount the qrcode element
+
+Next, we need to mount the qrcode element to the DOM.
+
+```js
+element.mount('qrcode');
+```
+
+This function will append the qrcode element to your div with an id `qrcode` as created in Step 3. **Ensure that there are no other elements in the document with the same id**.
+
+The **element should only be mounted once** in a single payment flow.
+
+### 6. Add an `onReady` event listener to handle events when the element is mounted
+
+```js
+window.addEventListener('onReady', (event) => {
+  /*
+    ... Handle event
+  */
+  window.alert(event.detail);
+});
+```
+
+This can be used to set a loading state as the checkout screen is being prepared.
+
+### 7. Add an `onSuccess` event listener to handle events when the payment is successful.
+
+```js
+window.addEventListener('onSuccess', (event) => {
+  /*
+    ... Handle event on success
+  */
+  window.alert(event.detail);
+});
+```
+
+### 8. Add an `onError` event listener to handle events when the payment has failed.
+
+```js
+window.addEventListener('onError', (event) => {
+  /*
+    ... Handle event on error
+  */
+  console.log(event.detail);
```

Comment:
why use `window.alert` above, but  use `console.log` here

Reply:
yeah, changed the alert to console